src/entities/Grant/constants.ts

```diff
@@ -3,10 +3,28 @@ import { GrantTierAttributes, GrantTierStatus, GrantTierType } from './types'
 export const GRANT_PROPOSAL_DURATION_IN_SECONDS = 1209600
 export const GRANT_PROPOSAL_MIN_BUDGET = 100
 export const GRANT_PROPOSAL_MAX_BUDGET = 240000
+export const MAX_LOWER_TIER_GRANT_FUNDING = 20000
+
+export const GRANT_PROPOSAL_MIN_PROJECT_DURATION = 1
+export const GRANT_PROPOSAL_MAX_PROJECT_DURATION = 12
+export const MIN_LOW_TIER_PROJECT_DURATION = GRANT_PROPOSAL_MIN_PROJECT_DURATION
```

Comment:
I'd add GRANT_PROPOSAL_ to all these new variables. Or maybe drop that from the other ones 🤔